src/components/ThemeSwitcher.tsx

```diff
@@ -4,6 +4,12 @@ import { IconButton, Tooltip } from '@radix-ui/themes'
 
 export function ThemeSwitcher() {
   const theme = useTheme()
+  // @ts-expect-error we have commonjs set as module option
+  const isDev = import.meta.env.DEV
```

Comment:
Would `process.env.NODE_ENV === 'development'` work here?

Reply:
Thanks for the suggestion. I was under the impression it wasn't available in the render process 👀. Replaced with `process.env.NODE_ENV === 'development'` 👍